<% if object.errors.any? %>
  <div id='error_explanation'>
    <div class='alert alert-danger'>
      <%= pluralize(object.errors.count, 'つ') %>のエラーがあります。
    </div>
    <ul>
      <% object.errors.full_messages.each do |msg| %>
        <li><%= msg %></li>
      <% end %>
    </ul>
  </div>
<% end %>
